src/main/java/org/betterx/betterend/entity/SilkMothEntity.java

```diff
@@ -322,4 +329,17 @@ public void tick() {
             }
         }
     }
+
+    @Override
+    public boolean isFood(ItemStack itemStack) {
+        if (!(itemStack.getItem() instanceof BlockItem)){
+            return false;
+        }
+        return ((BlockItem) (itemStack.getItem())).getBlock() instanceof EndPlantBlock;
```

Comment:
#103 suggested flowers be used to breed moths. Given that moths don't actually seek out end flowers--and that there's no dedicated list of end "flowers", I just used any end vegetation. Seemed like a least-effort solution, but I could see either:

- Creating a hard-coded list of "end flowers"
- Dynamically determining whether an end plant is a flower based on whether it can be crafted into dye
- Giving silk moths a dedicated breeding item (maybe giving a purpose to an end item that doesn't have one right now?)
- Letting silk moths be bred by any _overworld_ flower instead (cool lore implications?) / as well